README.md

```diff
@@ -38,29 +38,23 @@ height="80" />](https://f-droid.org/packages/io.github.muntashirakon.AppManager)
 alt="Get it on IzzyOnDroid"
 height="80" />](https://apt.izzysoft.de/fdroid/index/apk/io.github.muntashirakon.AppManager)
 
-App Manager Docs: https://muntashirakon.github.io/AppManager
-
-Telegram Support Group: https://t.me/AppManagerAndroid
-
-Telegram Update Channel: https://t.me/AppManagerChannel
-
-Follow **@AMUpdateChannel** on Twitter: https://twitter.com/AMUpdateChannel
+[App Manager Docs](https://muntashirakon.github.io/AppManager)
 
 ### Translations
 
-Translate **App Manager** at _Weblate_: https://hosted.weblate.org/engage/app-manager/
+Help translate [App Manager at Hosted Weblate](https://hosted.weblate.org/engage/app-manager/)
+The App Manager Docs are translated [at Crowdin](https://crwd.in/app-manager-docs) till https://github.com/MuntashirAkon/AppManager/issues/303 is closed.
 
-Translate **App Manager Docs** at _Crowdin_: https://crwd.in/app-manager-docs
 
 [![Translation status](https://hosted.weblate.org/widgets/app-manager/-/multi-auto.svg)](https://hosted.weblate.org/engage/app-manager/)
 
 ### Mirrors
 
-**GitLab**: https://gitlab.com/muntashir/AppManager
+**[GitLab](https://gitlab.com/muntashir/AppManager)**
 
-**Riseup**: https://0xacab.org/muntashir/AppManager
+**[Riseup](https://0xacab.org/muntashir/AppManager)**
 
-**Codeberg**: https://codeberg.org/muntashir/AppManager
+**[Codeberg](https://codeberg.org/muntashir/AppManager)**
```

Comment:
Since they're just a single word, making them inline would look better. Something like

**[GitLab](https://gitlab.com/muntashir/AppManager)** - **[Riseup](https://0xacab.org/muntashir/AppManager)** - **[Codeberg](https://codeberg.org/muntashir/AppManager)**

---
We also have mirror at Sourcehut but not displayed for now.